support-metrics-client/src/main/java/io/confluent/support/metrics/SupportedServerStartable.java

```diff
@@ -59,7 +58,7 @@ public SupportedServerStartable(Properties brokerConfiguration) {
 
         metricsReporter =
             new MetricsReporter(server, brokerConfigurationPlusMissingPSSettings, serverRuntime);
-        metricsThread = Utils.daemonThread("ConfluentProactiveSupportMetricsAgent", metricsReporter);
+        metricsThread = newThread("ConfluentProactiveSupportMetricsAgent", metricsReporter, true);
```

Comment:
Yes, the `utils` package is internal. Not ideal that we don't make that very clear. The public API packages are the ones that can be found in the published Javadoc.